Sources/Views/PricingLanding.swift

```diff
@@ -286,20 +319,20 @@ private func planCost(_ cost: PricingPlan.Cost) -> Node {
   )
 }
 
-private func pricingPlan(_ plan: PricingPlan) -> ChildOf<Tag.Ul> {
+private func pricingPlan(subscriberState: SubscriberState, plan: PricingPlan) -> ChildOf<Tag.Ul> {
   let cost = plan.cost.map(planCost) ?? div([])
 
-  let ctaButton = a(
-    [
-      href("#"),
-      `class`([
-        Class.margin([.mobile: [.top: 2], .desktop: [.top: 3]]),
-        plan.cost == nil ? contactusButtonClasses : choosePlanButtonClasses
-        ])
-    ],
-    [
-      plan.cost == nil ? "Contact Us" : "Choose plan"
-    ]
+  let ctaButton = subscriberState.isActive
```

Comment:
now we're removing the action buttons in the plans when you are an active subscriber:

<img width="1084" alt="image" src="https://user-images.githubusercontent.com/135203/63179639-96036b80-c01a-11e9-9d30-3efda4b46abb.png">